Packs/Slack/ReleaseNotes/1_3_7.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Slack v2
+- Fixed an issue where exception in invite to mirrored channel function was not caught.
+- Upgraded the Docker image to demisto/slack:1.0.0.12410.
```

Comment:
No need for the docker note. Either replace with `Upgraded slack API version` or `Stability improvements`

Reply:
This is a new content convention, docker image upgrades must be explicitly mentioned

Reply:
Done X2